### ISSUE TITLE
Lotek Receivers with issue: "Missing antenna dongle type"

### ISSUE THREAD

**Issue by leberrigan** [MotusDev/Motus-TO-DO#476]
Deployments for Lotek receivers are triggering the issue "Missing antenna dongle type" when this not even an option when entering deployment information for this type of receiver. Example here: https://motus.org/data/issues?id=64#v_sensor_antenna_dongle

Lotek receivers do not use radio dongles.

**Comment by cjardine-bsc** [comment]
We decided we wanted dongle type to be a required field. 

We should decide what we'd like there for Lotek receivers. Our current options are: 

CTT LifeTag (TM)
Funcube Pro
Funcube; Unconfirmed
RTL-SDR
XCVR

If none of those make sense for lotek we will need to add to the list. 

Changes needed: 

1. Update existing lotek deployments to the correct dongle
2. Change deployment page to populate dongle for lotek receivers

**Comment by Hussell** [comment]
> Lotek receivers do not use radio dongles.

So null seems like the correct dongle-type, and the "Missing Antenna Dongle Type" error should not be generated for Lotek receivers. That only requires a change to prDataValidation_pr.

**Comment by cjardine-bsc** [comment]
Not if we want a not null constraint on the dongle type column. For your mapping products your requested that this column not include NULLs. Do you no longer need that constraint in place?

**Comment by cjardine-bsc** [comment]
Also note that the validation proc gives this error for antenna deployments with dongle_type = "Funcube unconfirmed" Once the dongle type is updated they will no longer be flagged by the validation proc. So this does not require any changes to that proc. 

For the moment I've set Lotek receivers to dongle_type = NULL so that the validation proc wont flag them. But I'd like to resolve whether this column needs to be required or not before we close this issue.

**Comment by Hussell** [comment]
Initially null was displayed as one of the dongle-types in the drop-down list on the map, Stu requested it be removed from the list (i.e. display Lotek receivers when showing dongle-type=All, but don't allow dongle-type=null as a display option), which I did. Then dongle-type got removed from the map interface completely, rendering the whole thing moot.

So, no, there's no need for a not-null constraint on the dongle_type column due to the map products.
